config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
```

Comment:
- [ ] Nicely done, kindly check I had Issues with `cors` while testing your project, I had to un-comment the code in this `cors.rb` file and `gem "rack-cors"` in your gem file to make it work.

Reply:
@danielufeli thank you so much. What exactly am I suppose to do? I am lost.